Polar/cube.geojson

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
Please adjust the cube.geojson according to the finalized cube specification. 
Thanks!